extensions/wikia/WikiFactory/SpecialWikiFactory_ajax.php

```diff
@@ -639,16 +629,24 @@ function axWFactorySaveVariable() {
 
 	}
 
-	if (empty($form_id)) $div_name = "wf-variable-parse"; else $div_name = "wf-variable-parse-{$form_id}";
+	$div_name = 'wf-variable-parse' . ( empty( $form_id ) ? '' : "-$form_id" );
+
+	if ( isset( $cv_value, $cv_variable_type ) ) {
+		$value = WikiFactory::parseValue( $cv_value, $cv_variable_type );
+		$value = htmlspecialchars( $value );
+	} else {
+		$value = '';
+	}
 
 	return json_encode(
-		array(
+		[
 			"div-body" => $return,
 			"is-error" => $error,
 			"tag-name" => $tag_name,
 			"tag-wikis" => $tag_wiki_count,
 			"div-name" => $div_name,
-		)
+			'variable-value' => $value,
```

Comment:
Add this to the response so we can update the variable form (e.g. if a value was updated from `TRUE` to `FALSE`, actually change the UI to show `FALSE` rather than make the user reload the page to confirm.)